rosetta/offline_config_devnet.toml

```diff
@@ -0,0 +1,5 @@
+# OfflineSettings section of the rosetta server offline mod
+[OfflineSettings]
+    ChainID = "D"
+    MinGasPrice = 50000
```

Comment:
should be switched (min gas limit 50k, min gas price 1000000000)

Reply:
done.